src/components/VmStatus/VmStatus.js

```diff
@@ -160,15 +200,38 @@ export const VmStatus = ({ vm, launcherPod, importerPods, migration, verbose })
   }
 };
 
+VmStatusDetail.propTypes = {
+  vm: PropTypes.object.isRequired,
+  launcherPod: PropTypes.object,
+  statusDetail: PropTypes.object.isRequired,
+  verbose: PropTypes.bool,
+};
+
+VmStatusDetail.defaultProps = {
+  verbose: false,
+  launcherPod: undefined,
+};
+
+export const VmStatus = ({ vm, launcherPod, cdiPods, migration, dataVolumes }) => {
+  const statusDetail = getVmStatusDetail(vm, launcherPod, cdiPods, migration, dataVolumes);
+  return (
+    <div id={`kubevirt-vm-status-${getNamespace(vm)}/${getName(vm)}`}>
```

Comment:
please use `utils/getId` function. Also I would rather avoid using `/` character inside an id (https://stackoverflow.com/questions/70579/what-are-valid-values-for-the-id-attribute-in-html).

Maybe you can refactor the `getId` function to use `_` instead of `-` to not have only hyphens in our ids. I guess you could also add a check for empty namespace while you are at it.